### PR TITLE
Card cli - restarted the branch

### PR DESCRIPTION
**Pull request by GerryG**
To run CI

### CODE REVIEW

File: card/lib/card/config/environments/test.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- encoding : utf-8 -*-
-Decko.application.class.configure do
+Card::Application.configure do
```
Comment (GerryG) [inline card/lib/card/config/environments/test.rb:2]: This variation loads, but even though it looks like seed -t is loading fixtures, no test data (load of joe user fails in early test setup). I think this pattern works.

**Review by ethn — COMMENTED** [review]
A lot of good work here. Tomorrow I'll write PK and get him to weigh in.

**Comment by ethn** [comment]
Is this branch still viable?  Do we keep the PR open?

**Comment by ethn** [comment]
I think we've moved on to other branches, right?

**Pull request closed without merging**